src/containers/MainPanel.js

```diff
@@ -0,0 +1,137 @@
+import React, {Component} from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Button from '../components/Button'
+import Input from '../components/Input'
+import TextArea from '../components/TextArea'
+
+import { isEmpty } from 'lodash'
+
+import { 
+  createDocument,
+  updateDocument,
+  removeDocument,
+  changeTitleInput,
+  changeTextInput,
+} from '../reducers/documents'
+
+import DelIcon from '../assets/images/delete.png'
+import SaveIcon from '../assets/images/pencil.png'
+
+import './css/Form.css'
+
+class MainPanel extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      text: '',
+    };
+  }
+
+  changeTitleInput = (event) => {
+    this.props.changeTitleInput(event.target.value)
+  }
+
+  changeTextInput = (event) => {
+     this.props.changeTextInput(event.target.value)
+  }
+
+  handleSave = () => {
+    if (this.props.titleInput.length > 1) {
+      if(this.props.mode === 'add') {
+        this.props.createDocument({
+          id: this.props.documents.length,
```

Comment:
don't use length for id, generate id somehow (you can use timestamp for example)